pkg/agent/core/ngt/service/ngt.go

```diff
@@ -93,7 +92,7 @@ type ngt struct {
 	indexing  atomic.Value
 	saving    atomic.Value
 	cimu      sync.Mutex // create index mutex
-	lastNoice uint64     // last number of create index execution this value prevent unnecessary saveindex.
+	lastNocie uint64     // last number of create index execution this value prevent unnecessary saveindex.
```

Comment:
Does it suppose `lastNotice`? :thinking:

Reply:
lastNotice is typo.

as you can see in the same struct field  nocie.
NOCIE means  (Number Of Create Index Execution)

so lastNocie also same

lastNocie  = Last Number Of Create Index Execution

Reply:
Got it !